### PR TITLE
Another Batch of Permissions

### PR DESCRIPTION
**Pull request by FoseFx**
## Which issues does this pull request close?
progresses #877 

## [OPTIONAL] Give testing instructions to reviewers
- ProperySet
- PropertyValue
- Bed
- TaskTemplates

### CODE REVIEW

**Comment by FoseFx** [comment]
WIth Max being OOO, can you take a look at this?

**Comment by PaulKalho** [comment]
> WIth Max being OOO, can you take a look at this?

I will take a look :)

File: spicedb/_version [hunk 1]
```diff
@@ -1 +1 @@
-2
+3
```
Comment (FoseFx) [inline spicedb/_version:1]: remind me to bump this

File: services/tasks-svc/internal/bed/bed.go [hunk 2]
```diff
@@ -161,6 +160,22 @@ func (ServiceServer) GetBedByPatient(
...
+	// check permissions
+	user := commonPerm.UserFromCtx(ctx)
+	checks := make([]hwauthz.PermissionCheck, 0)
+	checks = append(checks,
+		hwauthz.NewPermissionCheck(user, patientPerm.PatientCanUserGet, patientPerm.Patient(patientId)))
+
+	if result != nil {
+		checks = append(checks,
+			hwauthz.NewPermissionCheck(user, perm.BedCanUserGet, perm.Bed(result.BedID)),
+			hwauthz.NewPermissionCheck(user, roomPerm.RoomCanUserGet, roomPerm.Room(result.RoomID)),
```
Comment (PaulKalho) [inline services/tasks-svc/internal/bed/bed.go:172]: wouldn't be the check for `RoomCanUserGet` enough, as it implies (through transitivity) that the `bed` can also "be get"?

File: spicedb/task_template.test.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  - "ward:a#organization@organization:a"
+  - "ward:b#organization@organization:b"
+
+  # task_template A belongs to ward A,
+  # and template B to user Bob
+  - "task_template:a#owner@ward:a"
+  - "task_template:b#owner@user:bob"
+
+tests:
+  - name: If the owner is a user, only the owner of a template can RUD it
```
Comment (PaulKalho) [inline spicedb/task_template.test.yaml:29]:
```suggestion
  - name: If the owner is a user, only the owner of a template can GUD it
```

**Review by PaulKalho — APPROVED** [review]

File: services/tasks-svc/internal/bed/bed.go (hunk 2 above)
Comment (FoseFx) [reply services/tasks-svc/internal/bed/bed.go:172]: yes, currently that's true. but in the future we maybe have different roles, some may not have access to certain beds, and thus should not see them here

**Pull request merged**